docs/references.md

```diff
@@ -1,11 +1,48 @@
 # References
 
+## 2024 - Pass The Salt (PTS) Workshop
+### [KubeHound: Identifying attack paths in Kubernetes clusters at scale with no hustle](https://cfp.pass-the-salt.org/pts2024/talk/WA99YZ/) 
+
+[Slides :fontawesome-solid-file-pdf:{ .pdf } ](files/PassTheSalt24/Kubehound - PassTheSalt - slides.pdf){ .md-button } [Jupyter notebook :fontawesome-brands-python:{ .python } ](https://github.com/DataDog/KubeHound/tree/main/deployments/kubehound/notebook/KubehoundDSL_101.ipynb){ .md-button }
```

Comment:
Hmm idk but the formatting seems broken on my side (even on main, it is not specific to this PR but noted it):
<img width="948" alt="image" src="https://github.com/DataDog/KubeHound/assets/66471981/1d7c1ac5-3d52-4b48-94aa-2b74ad4db5df">

Reply:
did you use the local command to render it ? `make local-wiki`

Reply:
Nope, I thought it was some markdown syntax I didn't know about that would be understood by the github website.
If it's expected to be read locally, testing the local wiki but should be good :D